### DISCUSSION TITLE
Gecko linux isos updation dec 2023

### DISCUSSION THREAD

**Discussion by praveens2208** [community/community#80972] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Gecko linux  isos not updated since 2022 August is the rolling release discontinued

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @praveens2208! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!
